modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/NavigationPanel.js

```diff
@@ -23,7 +24,7 @@ const ITEM_TYPES_SYMBOL = {
 	folder: 'folder',
 };
 
-export default function NavigationPanel({items, selectedItemId}) {
```

Comment:
Why remove the `selectedItemId`?
Is need to expand the three around the current node

Reply:
I updated the code to cover this case again, sorry because from the jira description I understood it was not needed anymore